src/neuroconv/datainterfaces/ecephys/cellexplorer/cellexplorerdatainterface.py

```diff
@@ -25,7 +25,31 @@ def __init__(self, file_path: FilePathType, verbose: bool = True):
 
         hdf5storage = get_package(package_name="hdf5storage")
 
-        super().__init__(spikes_matfile_path=file_path, verbose=verbose)
+        file_path = Path(file_path)
+
+        # Temporary hack to get sampling frequency from the spikes cellinfo file until next SI release.
+        import h5py
+
+        try:
+            matlab_file = scipy.io.loadmat(file_name=str(file_path), simplify_cells=True)
+            if "spikes" not in matlab_file.keys():
+                raise ValueError(f"Matlab file  {file_path} does not contain `spikes` field")
```

Comment:
Yeah this is definitely more informative; minor change to type though, this is still a KeyError right?

```suggestion
                raise KeyError(f"CellExplorer file '{file_path}' does not contain 'spikes' field.")
```

Reply:
Correct about KeyError.